data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg

```diff
@@ -0,0 +1,1335 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Also, there are some typos in the file names, `con.github.PlugaruT.wingpanel-monitor.svg` should be `com.github.plugarut.wingpanel-monitor.svg`

Reply:
Sure.